contracts/modules/utils/RequireUtils.sol

```diff
@@ -61,11 +77,100 @@ contract RequireUtils {
             )
           )
         )
-      ) == _wallet, "RequireUtils#requireConfig: UNEXPECTED_COUNTERFACTUAL_IMAGE_HASH");
+      ) == _wallet, "RequireUtils#publishConfig: UNEXPECTED_COUNTERFACTUAL_IMAGE_HASH");
     }
 
     // Emit event for easy config retrieval
     emit RequiredConfig(_wallet, imageHash, _threshold, abi.encode(_members));
+
+    if (_index) {
+      // Register last event for given wallet
+      lastWalletUpdate[_wallet] = block.number;
+    }
+  }
+
+  function publishSigners(
```

Comment:
Maybe we could call this `publishInitialConfig()` instead to make it clear that it's just for the initial config that we need this.

Reply:
I think the comment is outdated